src/commands/html-proofer.yml

```diff
@@ -167,7 +163,6 @@ steps:
           --directory-index-file << parameters.directory-index-file >> \
           <<# parameters.disable-external >> --disable-external <</ parameters.disable-external >> \
           <<# parameters.empty-alt-ignore >> --empty-alt-ignore <</ parameters.empty-alt-ignore >> \
-          --error-sort << parameters.error-sort >> \
```

Comment:
To maintain backward compatibility, you should add a boolean parameter, e.g.

```
  include-error-sort:
    description: "adds the --error-sort flag to the command"
    type: boolean
    default: true
 ```
 
 and then update this line,    `<<# parameters.include-error-sort >>--error-sort << parameters.error-sort >> <</ parameters.include-error-sort >> /`
 
This will not change existing usage for any user but would allow you to disable this feature.

Reply:
Better yet, we can leave the default error-sort parameter as blank and use that to evaluate the boolean.

```yaml
  error-sort:
    description:  "Defines the sort order for error output. Can be `:path`, `:desc`, or `:status` (default: `:path`)."
    type: string
    default: ""

#...

          <<# parameters.error-sort >>--error-sort << parameters.error-sort >> <</ parameters.error-sort >> \

```

Reply:
Fixed in commit 2ce2d2917c0e51102d73cacf98a2b4113c724361.